contents/docs/tutorials/feature-flags.md

```diff
@@ -0,0 +1,206 @@
+* Show advanced options by default to users in the 'Organization Admins' cohort who have `@bigcompany.com` in their `email` property
+
+**Roll out to a percentage of users**
+
+This option can be used either by itself or in combination with the property/cohort filters.
+
+When used by itself, it means that `X%` of your users will have the flag on and the other `(100-X)%` will have it off.
+
+This mechanism is _random but deterministic_. What this means is that it is a weighted random selection that establishes which users will have the flag on or off, but a given flag _will persist_ for a every user (if nothing else changes). This ensures that users don't have a different experience each time they visit your app.
+
+When used in conjunction with property filters, PostHog will first check if the user matches all the filters, and then apply the rollout percentage. This means that `X%` of the users who match the filters will see the flag, rather than `X%` of overall users.
+
+
+<div class='note-block'><b>Technical Note:</b> Our rollout percentage mechanism works by hashing the distinct ID of a user with the flag key, casting it to an integer, and dividing it by 100. That gives us a number between 0 and 1 which we compare to the rollout percentage value (also between 0 and 1) to determine if a user should have the flag on or off. This ensures an acceptable degree of randomness and accuracy while allowing flags to persist.</div>
+
+
+### Step 4: Saving Your Flag
+
+Just click 'Save feature flag' and that's it! Your flag will now be active. Clicking on it will also allow you to edit it as you wish.
+
+## Implementing the Feature Flag
+
+When you create a feature flag, PostHog displays an example snippet that you can plug into your codebase. It looks something like this:
+
+```js
+if (posthog.isFeatureEnabled('new-beta-feature')) {
+    // do something
+}
+```
+
+This snippet refers to our [JavaScript Integration](/docs/integrations/js-integration), which is also what we will be using for this tutorial. However, you can also use feature flags via our [API](/docs/api/overview) as well as other libraries, such as our [Python Integration](/docs/integrations/python-integration) (we're working to add this functionality to as many of our libraries as possible). 
+
+With that snippet, you can then do whatever you wish inside of it. You might change the CSS of a button, hide an entire section, or move things around. A simple example implementation would be:
+
+```js
+// Header text is blue by default
+// If flag is on, make the text red
+if (posthog.isFeatureEnabled('red-header-text')) {
+    document.getElementById('header-text').style['color'] = 'red'
+}
+```
+
+While this is a trivial example, you can have as much logic as you want being powered by the feature flag. You can also combine flags with `OR` & `AND` operations to create more complex logic.
+
+## Advanced Controls
+
+### Loading Flags
```

Comment:
Could we make this header more use-case driven? 'Loading Flags' doesn't make immediate sense to me why this helps me